public/app/features/canvas/runtime/scene.tsx

```diff
@@ -431,12 +430,12 @@ export class Scene {
       // @TODO Figure out click-drag functionality without phantom mouseup issue
       // https://github.com/daybrush/moveable/issues/481
 
-      // if (event.isDragStart) {
-      //   event.inputEvent.preventDefault();
-      //   setTimeout(() => {
-      //     this.moveable!.dragStart(event.inputEvent);
-      //   });
-      // }
+      if (event.isDragStart) {
+        event.inputEvent.preventDefault();
+        setTimeout(() => {
+          this.moveable!.dragStart(event.inputEvent);
+        });
+      }
```

Comment:
Strange this is "just working" now when it wasn't before (due to phantom mouse down issue), but we'll take it haha

Just a few minor things to clean up:

1. when clicking and dragging the cursor doesn't update to be "grabbing" element

https://user-images.githubusercontent.com/22381771/181291090-79e62867-7abf-44bf-922e-974090c59c69.mov


2. MINOR - cursor behavior when dragging element underneath is a little wonky

https://user-images.githubusercontent.com/22381771/181291160-9f2aa8f5-3236-471c-b3d7-e18da0043740.mov

Reply:
For part 2, I need to turn off pointer events for all other elements while dragging, then turn them back on when dragging is done. Need to find an efficient way to apply style to all OTHER elements.